arxiv/forms/csrf.py

```diff
@@ -116,7 +111,12 @@ def validate_csrf_token(self, form: 'CSRFForm', field: Field) -> None:
         """Validate the CSRF token passed with form data."""
         if field.data is None:
             raise ValidationError('Missing CSRF token')
-        digest, expires = self._split(field.data)
+
+        parts = field.data.split('::',1)
+        if len(parts) != 2:
+            raise ValidationError('CSRF token is malformed')
+
+        digest, expires = parts
```

Comment:
I'm curious, is this new check due to observed corruption in the token?

Reply:
I'm not sure how this wasn't caught before since it was making a unit test fail.